src/java/org/pantsbuild/tools/junit/impl/SpecParser.java

```diff
@@ -94,9 +98,10 @@
       throw new SpecException(specString,
           String.format("Error linking %s.", className), e);
       // See the comment below for justification.
-    } catch (RuntimeException e) {
+    } catch (Exception e) {
       // The class may fail with some variant of RTE in its static initializers, trap these
-      // and dump the bad spec in question to help narrow down issue.
+      // and dump the bad spec in question along with the underlying error message to help
+      // narrow down issue.
```

Comment:
Does `some variant of RTE` mean RTE's subclasses? If so, `RuntimeException` should still apply right?

Reply:
I feel like there was a reason I widened it, but I don't remember it and it's probably fine without.